fusor-ember-cli/app/mixins/discovered-host-route-mixin.js

```diff
@@ -0,0 +1,28 @@
+import Ember from 'ember';
+
+export default Ember.Mixin.create({
+
+  setupController: function(controller, model) {
+    controller.set('model', model);
+    if (this.modelFor('deployment').get('isNotStarted')) {
+        controller.set('isLoadingHosts', true);
+        this.store.find('discovered-host').then(function(results) {
+          controller.set('allDiscoveredHosts', results.filterBy('is_discovered', true));
```

Comment:
this line is the actually fix. It was 
`controller.set('allDiscoveredHosts', results);`

but while I was at it, I DRYed up the code and made a mixin since the same code is shared by Engines and Hypevisors.